lib/ricohapi/mstorage/client.rb

```diff
@@ -90,7 +92,17 @@ def delete(media_id)
 
       # PUT /media/{id}/meta/user/{key}
       def add_meta(media_id, user_meta)
-        # TODO: do something
+        raise Error.new("user_meta is empty: nothing to request.") if user_meta.empty?
+        user_meta.each do |k, v|
+          validate(v)
+          if k.empty?
+            raise Error.new("Invalid parameter: One of the given keys is empty")
+          end
```

Comment:
This method will solve what I was worried about after the last review!
Thank you!